docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -52,32 +53,45 @@ include::_attributes.adoc[] <3>
 +
 <1> Set the `id` value to be the same as the file name but without the extension. You can shorten this if the file name is too long.
 <2> For information about how to create a good title for each content type, see xref:{doc-guides}/doc-reference.adoc#titles-and-headings[Titles and headings] on the "Quarkus style and content guidelines" page.
-<3> The `_attributes.adoc` include is required to ensure that attributes get resolved, the table of contents is generated, and content renders in the website portal. 
+<3> The `_attributes.adoc` include is required to ensure that attributes get resolved, the table of contents is generated, and content renders in the website portal.
 <4> Set at least one category to ensure that the content is findable on the link:https://quarkus.io/guides/[Quarkus documentation home page]. For a list of Quarkus categories, see xref:{doc-guides}/doc-reference.adoc#document-attributes-and-variables[Document attributes and variables] on the "Quarkus style and content guidelines" page.
 +
 [IMPORTANT]
 ====
 Ensure there are no line breaks in the header section until after `:categories:` line.
 ====
+
+. Add an abstract that describes the purpose of the guide.
 +
-. Add an abstract to describe the purpose of the guide.
 [IMPORTANT]
 ====
-The first sentence of the abstract must explain the value and some benefit of the content in less than 27 words because this automatically displays on the link:https://quarkus.io/guides/[Quarkus guides homepage]. 
+The first sentence of the abstract must explain the value and some benefit of the content in less than 27 words because this automatically displays on the link:https://quarkus.io/guides/[Quarkus guides homepage].
 There must also be a line break before and after the abstract.
 ====
 
+. For how-to and tutorial topics, include a set of prerequisites. For example:
```

Comment:
I think this level of detail and example would be good to add to the guidance that's in the existing [how-to](https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/_templates/template-howto.adoc) and [tutorial](https://raw.githubusercontent.com/quarkusio/quarkus/main/docs/src/main/asciidoc/_templates/template-tutorial.adoc) doc templates.  

The current how-to template doesn't mention prereqs at all.